app/src/pages/inside/dashboardPage/dashboardPage.jsx

```diff
@@ -213,14 +218,18 @@ export class DashboardPage extends Component {
             filter={filter}
             onFilterChange={onFilterChange}
           />
-          <DashboardList
-            dashboardItems={dashboardItems}
-            gridType={gridType}
-            userInfo={userInfo}
-            onDeleteItem={this.onDeleteDashboardItem}
-            onEditItem={this.onEditDashboardItem}
-            onAddItem={this.onAddDashboardItem}
-          />
+          {dashboardItems.length > 0 ? (
```

Comment:
Please add filter existence check, because in case project doesn't have any dashboards, there should be another view.

Reply:
Yout right! Thank you! I've added `filter` condition.